debezium-testing/debezium-testing-openshift/pom.xml

```diff
@@ -0,0 +1,313 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <parent>
+    <groupId>io.debezium</groupId>
+    <artifactId>debezium-testing</artifactId>
+    <version>1.1.0-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>debezium-testing-openshift</artifactId>
```

Comment:
I suppose this module is excluded from deployment to Maven Central?

Reply:
@gunnarmorling Does it need to be excluded?

Reply:
Not strictly. So far I just perceived this as something that we'd only use ourselves, in particular we wouldn't want to commit to "supporting" any external users. So I wouldn't see the need for deploying it. But if it makes things somehow easier, then I don't see a big problem with deploying it. After all, it's not covered in the documentation.